go.mod

```diff
@@ -0,0 +1,4 @@
+module github.com/tiny-go
```

Comment:
please use the correct module name: `github.com/metrico/micro-gc`

Reply:
@akvlad  Thank you for bringing it to my attention. It has been fixed in the new commit.